.github/workflows/test.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     strategy:
       matrix:
         go-version: [1.18.x, 1.23.x, 1.24.x]
-        platform: [ubuntu-20.04, ubuntu-22.04, ubuntu-24.04, windows-latest, macos-13, macos-14]
+        platform: [ubuntu-20.04, ubuntu-22.04, ubuntu-24.04, windows-latest, macos-13, macos-14, macos-15]
```

Comment:
nit: maybe it's time to remove ubuntu-20.04 as it is going to be sunset in a week (see https://github.com/actions/runner-images/issues/11101).

Reply:
Ah, it's done already later in this PR.

Reply:
Ah, yes, it's actually why I started this work; I wanted to remove that ticking time-bomb before tagging the `atomicwriter` module 😅